packages/turf-interpolate/index.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="geojson" />
+
+import {Points} from '@turf/helpers';
+
+/**
+ * http://turfjs.org/docs/#interpolate
+ */
+declare function interpolate(points: Points, cellSize: number, property?: number, units?: string, weight?:number): Points;
```

Comment:
Contradicts my previous comment, but `units?: string` could use `Units` instead of `string`.

```javascript
import {Units} from '@turf/helpers'
```